gui/builtinViewColumns/state.py

```diff
@@ -68,11 +68,17 @@ def getImageId(self, stuff):
                                                                 "gui")
         elif isinstance(stuff, Fit):
             fitID = self.mainFrame.getActiveFit()
-            projectionInfo = stuff.getProjectionInfo(fitID)
 
-            if projectionInfo is None:
+            # can't use isinstance here due to being prevented from importing CommandView.
+            #  So we do the next best thing and compare Name of class.
+            if type(self.fittingView).__name__ == "CommandView":
+                info = stuff.getCommandInfo(fitID)
+            else:
+                info = stuff.getProjectionInfo(fitID)
+
+            if info is None:
                 return -1
-            if projectionInfo.active:
+            if info.active:
```

Comment:
Well it's no longer projection info. It can be command info now, so it needs to be more generic. It's a variable scoped to this function, so there shouldn't be any issue in determining the meaning of it.